src/views/publish/upload-lookup.ejs

```diff
@@ -0,0 +1,62 @@
+<%- include("../partials/header", t); %>
```

Comment:
Don't need to pass t

Reply:
I think most of the header partials pass in t... Again chalk it up as tech debt and we can clear up later